### PR TITLE
EWLJ-314 design more visible links

### PR DESCRIPTION
**Pull request by shaal**
## JIRA Ticket(s)
- [EWLJ-314: Design more visible links](https://issues.ama-assn.org/browse/EWLJ-314)

## Description:
According to the approved design on the ticket, I added to links inside articles a darker color and an underline.

## To Test:

- [ ] Switch your JOE SG2 branch to `feature/EWLJ-314-design-more-visible-links`
- [ ] Run gulp serve
- [ ] Enable local SG2 in your D8 project
- [ ] In another terminal run `drush @joe.local cr`
- [ ] Go to http://ama-joe.local/article/why-we-need-music-player-every-patient-room/2019-03
- [ ] Hover above links inside the article, and you should see the style of white text color with orange background

### CODE REVIEW

File: styleguide/source/assets/scss/01-atoms/_links.scss [hunk 1]
```diff
@@ -16,9 +16,13 @@ a,
   .joe__article p & {
     text-decoration: underline;
 
+    &:active,
     &:hover,
     &:focus {
-      color: $red-darker;
+      // color: $red-darker;
```
Comment (designerneil) [inline styleguide/source/assets/scss/01-atoms/_links.scss]: remove comment

File: styleguide/source/assets/scss/01-atoms/_links.scss [hunk 2]
```diff
@@ -16,9 +16,13 @@ a,
...
+      color: $white;
+      text-decoration: none;
+      background-color: $orange-darker;
```
Comment (designerneil) [inline styleguide/source/assets/scss/01-atoms/_links.scss]: the background color should be $blue-darkest (#29688a)

**Review by designerneil — DISMISSED** [review]

File: styleguide/source/assets/scss/01-atoms/_links.scss (hunk 1 above)
Comment (shaal) [reply styleguide/source/assets/scss/01-atoms/_links.scss]: comment removed

File: styleguide/source/assets/scss/01-atoms/_links.scss (hunk 2 above)
Comment (shaal) [reply styleguide/source/assets/scss/01-atoms/_links.scss]: background-color updated to $blue-darkest

**Review by tcfunk — APPROVED** [review]
Looks good 👍

**Pull request merged**